File: cpp/bc2582.cpp
#include <iostream>

using namespace std;

int main()
{
	string vec[] = {"PROXYCITY", "P.Y.N.G.", "DNSUEY!", "SERVERS", "HOST!", "CRIPTONIZE", "OFFLINE DAY", "SALT", "ANSWER!", "RAR?", "WIFI ANTENNAS"};
	int nCa, fVal, sVal, so;
	
	cin >> nCa;
	
	for(int i = 0; i < nCa; i++)
	{
		cin >> fVal >> sVal;
		so = fVal + sVal;
		cout << vec[so] << endl;
	}
	
}

File: python/bc2582.py
nCa = int(input())

al = ["PROXYCITY", "P.Y.N.G.", "DNSUEY!", "SERVERS", "HOST!", "CRIPTONIZE", "OFFLINE DAY", "SALT", "ANSWER!", "RAR?", "WIFI ANTENNAS"]

for c in range(0, nCa):
    vec = input().split(" ")

    fVal = int(vec[0])
    sVal = int(vec[1])
    so = fVal+sVal
    print(f"{al[so]}")
